### ISSUE TITLE
Translate draggable item to touch position

### ISSUE THREAD

**Issue by Neha1490** [h6ah4i/android-advancedrecyclerview#393]
When some item is selected for dragging it should be translated to finger touch position. I am using grid layout manager for recycler view.
How it can be achieved?

**Comment by h6ah4i** [comment]
@Neha1490 Hi. Unfortunately, this library does not support dragging multiple items at once. Also I do not know any other libraries which offers the feature you need 😞

Does anyone know how to implement this feature?

**Comment by Neha1490** [comment]
Hi
I think there is some miscommunication or misundersatnding about the question. Sorry for that.

I am dragging single item only at once.
When we long press on an item it gets scaled to the mid position.
What I want is to get the touch position of finger on long press and scale it and place it at finger position.

**Comment by h6ah4i** [comment]
@Neha1490 Hi. Sorry for my misunderstanding. I have fixed the issue in the following branch, so could you try it out through JitPack?

- [feature/improve_drag_effect_for_grid_layout](https://github.com/h6ah4i/android-advancedrecyclerview/tree/feature/improve_drag_effect_for_grid_layout)

```gradle
repositories {
    maven { url "https://jitpack.io" }
}
dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:0a90b7fb9b7bd0b4576bbbd883b010be64690799'
}
```

**Comment by h6ah4i** [comment]
Released in v0.11.0
